include/edm4hep/Constants.h

```diff
@@ -50,6 +50,14 @@ static constexpr const char* pidParameterNames = "ParameterNames";
 static constexpr const char* pidAlgoName = "AlgoName";
 static constexpr const char* pidAlgoType = "AlgoType";
 
+// Parameter names for Generator level metadata
+static constexpr const char* GeneratorToolVersion = "GeneratorToolVersion";
+static constexpr const char* GeneratorToolName = "GeneratorToolName";
+static constexpr const char* GeneratorToolDescription = "GeneratorToolDescription";
+static constexpr const char* GeneratorEventParameters = "GeneratorEventParameters";
+static constexpr const char* GeneratorPdfInfo = "GeneratorPdfInfo";
```

Comment:
These are now clashing with the generated classes. Maybe all of these should just be prefixed with `Gen` instead of `Generator`?

Reply:
I think I will do that indeed.

Reply:
Or we add a namespace for the constants.
And we copy the existing constants into the namespace and deprecate the namespaceless ones (we can even align spelling when we do that without breaking anything)

Reply:
that would be a major change and a bit cumbersome if dealing e.g. with the matrices end the corresponding enums. Do you really want to do that?

Reply:
IIRC the enums were supposed to stay where they are, since they aren't constants

Reply:
I would like a `edm4hep::label` namespace for things that are a label. the enums for covariance matrix access can / should stay in the `edm4hep` namespace.

Reply:
See #315 for a proposal with a new `labels` namespace.